Modules/Core/Common/wrapping/test/itkPointSetSerializationTest.py

```diff
@@ -52,7 +52,7 @@
 arr = itk.array_view_from_vector_container(v_point)
 points_vc = itk.vector_container_from_array(arr.flatten())
 
-point_set.SetPoints(points_vc)
```

Comment:
Can we make this "in addition" instead of "instead of" to improve coverage, i.e.,

```
point_set.SetPoints(points_vc)
point_set.SetPointsByCoordinates(points_vc)
```

Reply:
@thewtex I see your point; this pull request would take away code coverage from `PointSet::SetPoints(PointsVectorContainer *)`. But then again, I think we should eventually deprecate this specific `SetPoints` overload anyway, to avoid undefined behavior (issue #4848).

If we still want to have this specific `SetPoints` overload around for a while, I would prefer a separate isolated test for this specific overload.

Reply:
@N-Dekker yes, that is a good approach to both indicate that the code is unsafe and keep the coverage.